src/RestApiClient.js

```diff
@@ -14,7 +15,7 @@ class RestApiClient {
         'Content-Type': 'application/json',
         'Accept': 'application/json',
       },
-      credentials: overrides.credentials || 'same-origin',
+      credentials: 'same-origin',
```

Comment:
We didn't need to set this with `overrides.credentials`. If there is an override for this, it gets replaced when the objects are `merge()`ed in the follow up statement.